eventsourcing/src/test/java/org/axonframework/eventsourcing/eventstore/AggregateSequenceNumberPositionTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.eventsourcing.eventstore;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+/**
+ * Test for {@link AggregateSequenceNumberPosition}.
+ *
+ * @author John Hendrikx
+ */
+class AggregateSequenceNumberPositionTest {
+    private static final AggregateSequenceNumberPosition TEN = new AggregateSequenceNumberPosition(10);
```

Comment:
Super nit: pretty sure the style guide should put an empty line between the `class` reference and the first field/method inside the class block.

Reply:
I'll fix it, I never do it myself as I think the indent is clear enough already, but I get this comment from other people sometimes as well.  I do however always put empty lines between different types of fields (private/public) and in code between any code declaring a local and other code :)